grails-app/i18n/messages.properties

```diff
@@ -3322,7 +3322,9 @@ react.putAway.splitLine.label=Split line
 react.putAway.totalQty.label=Total QTY
 react.putAway.putAwayQty.label=Putaway QTY
 react.putAway.sumOfAll.label=Sum of all split items quantities cannot be higher than original putaway item quantity
+react.putAway.negativeSumOfAll.label=Items quantity cannot be lower than 1
```

Comment:
We don't have to do this in this PR, but we should change all language around higher vs lower to greater than vs less than.